lang/semgrep-grammars/src/semgrep-toml/grammar.js

```diff
@@ -0,0 +1,29 @@
+/*
+  semgrep-toml
+
+  Extends the standard toml grammar with semgrep pattern constructs.
+*/
+
+const base_grammar = require('tree-sitter-toml/grammar');
+
+module.exports = grammar(base_grammar, {
+  name: 'toml',
+
+  conflicts: ($, previous) => previous.concat([
+  ]),
+
+  /*
+     Support for semgrep ellipsis ('...') and metavariables ('$FOO'),
+     if they're not already part of the base grammar.
+  */
+  rules: {
+  /*
+    semgrep_ellipsis: $ => '...',
+
+    _expression: ($, previous) => choice(
+      $.semgrep_ellipsis,
+      ...previous.members
+    ),
+  */
+  }
```

Comment:
this won't currently support writing rules for toml, right? Obviously we don't need that for lockfile parsing, but wonder if we should make it obvious in a comment in semgrep-core when we consume this or something